src/urllib3/fields.py

```diff
@@ -76,7 +93,7 @@ def replacer(match):
     return result
 
 
-def format_header_param_html5(name, value):
+def format_header_param_html5(name: str, value: Union[str, bytes]) -> str:
```

Comment:
Should we use `_TYPE_FIELD_VALUE` here too, as we do in `RequestField.__init__`?